### ISSUE TITLE
result of fine-tuning on another dataset is odd

### ISSUE THREAD

**Issue by ohjoon2** [MhLiao/DB#254]
This is because I made mistake when parsing the label of dataset. So, close the issue.

Thank you for sharing nice work here!

~~I wonder why the result from fine-tuned(on [FUNSD](https://arxiv.org/pdf/1905.13538.pdf)) one is much worse than before(ic15-resnet50 pretrained model in this official repo).~~

~~Any body here fine-tuning on another dataset and suffered from similar results?~~
~~(Loss is still being converged, so I will report the results based on later epoch's model.)~~

### result from ic15 pretrained model from this repo
![20190122_181226](https://user-images.githubusercontent.com/45305396/118977235-e3132f00-b9b0-11eb-9939-e3e1e2f4647c.jpg)

### result from ic15 pretrained model + FUNSD dataset finetuning
![20190122_181226](https://user-images.githubusercontent.com/45305396/118977148-cecf3200-b9b0-11eb-96a3-bc9780e67b3a.jpg)

![DB-loss](https://user-images.githubusercontent.com/45305396/118978437-48b3eb00-b9b2-11eb-8b13-17bd5790048e.png)

**Comment by Kyle-Lai** [comment]
@ohjoon2 , hello, could you please tell me how to print the probability map and threshold map?

**Comment by ohjoon2** [comment]
> @ohjoon2 , hello, could you please tell me how to print the probability map and threshold map?

I draw binary map from binarized results in ```representer.py```

**Comment by ohjoon2** [comment]
This is because I made mistake when parsing the label of dataset. So, close the issue.
